src/renderer/migrations/hotbar-store.injectable.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { hotbarStoreMigrationsInjectionToken } from "../../common/hotbars/migrations";
+
+const hotbarStoreMigrationsInjectable = getInjectable({
+  id: "hotbar-store-migrations",
+  instantiate: () => undefined,
+  injectionToken: hotbarStoreMigrationsInjectionToken,
+});
```

Comment:
Requirement for these just highlights the fact that running migrations should be extracted from the stores in a way that `migrations` is concept of main. 

That would mean that we could have `runMigrations` runnable which is ran at `onLoadOfApplication`.